bricksrc/equipment.py

```diff
@@ -448,21 +468,35 @@
                     TAG.Heat,
                     TAG.Equipment,
                 ],
-                "parents": [BRICK.Hot_Water_System, BRICK.Water_Valve],
+                "parents": [BRICK.Water_Valve],
             },
         },
     },
     "Cooling_Valve": {"tags": [TAG.Valve, TAG.Cool, TAG.Equipment]},
     "Water_Valve": {
         "tags": [TAG.Valve, TAG.Water, TAG.Equipment],
+        SKOS.definition: Literal(
+            "Building water is provided by utility company and sourced through a master building feed."
+        ),
         "subclasses": {
             "Chilled_Water_Valve": {
                 "tags": [TAG.Chilled, TAG.Valve, TAG.Water, TAG.Equipment],
-                "parents": [BRICK.Chilled_Water_System],
             },
         },
     },
+    "Gas_Valve": {
+        "tags": [TAG.Gas, TAG.Valve, TAG.Equipment],
+        SKOS.definition: Literal(
+            "Building gas is provided by utility company and sourced through a master building feed."
+        ),
+    },
     "Isolation_Valve": {"tags": [TAG.Isolation, TAG.Valve, TAG.Equipment]},
+    "Steam_Valve": {
+        "tags": [TAG.Steam, TAG.Valve, TAG.Equipment],
+        SKOS.definition: Literal(
+            "Building steam is provided by the utility company and sourced through a master building feed."
```

Comment:
Same thing here: this is a definition for Steam, not a Steam Valve. In general we also can't assume that steam is provided by the utility company (it is often produced on site), so I would just remove this.